### PR TITLE
Improve quarkus platform documentation

### PR DESCRIPTION
**Pull request by carlesarnal**

### CODE REVIEW

File: platforms/quarkus/api/README.md [hunk 1]
```diff
@@ -13,18 +13,56 @@ mvn quarkus:dev
...
+### Build Options
+
+ - `-Pprod` enables Quarkus's *prod* configuration profile, which uses configuration options suitable for a production environment, 
+   e.g. a higher logging level.
+   
+
+##Runtime Configuration
+
+- In the *dev* mode, the application uses an in-memory H2 database`.
+- In the *prod* mode, you have to provide connection configuration for a PostgreSQL server as follows:
```
Comment (EricWittmann) [inline platforms/quarkus/api/README.md:33]: PostgreSQL or MySQL, right?  Both are supported?

Comment (EricWittmann) [reply platforms/quarkus/api/README.md:33]: If yes, then we should document how to configure the database type.

File: platforms/quarkus/api/README.md [hunk 2]
```diff
@@ -13,18 +13,56 @@ mvn quarkus:dev
...
+To see additional options, visit:
+ - [OIDC options](https://quarkus.io/guides/security-openid-connect) 
+ 
+ 
+
+
+Note that in order to have the full `apicurio-studio` app running both the `api` and the `ws` needs to share the same database.
+
 
 ## Creating a native executable
```
Comment (EricWittmann) [inline platforms/quarkus/api/README.md:60]: Let's drop this section - I don't think we're ever going to be building a native image.

Comment (EricWittmann) [reply platforms/quarkus/api/README.md:60]: Same for all 3 readme files.

File: platforms/quarkus/api/README.md (hunk 1 above)
Comment (EricWittmann) [reply platforms/quarkus/api/README.md:33]: Same change needed for the -ws image as well.

Comment (carlesarnal) [reply platforms/quarkus/api/README.md:33]: Everything should be ok with MySQL as well, just maybe some minor changes needed at config level. I'll check that and document how to configure the database type.

**Comment by EricWittmann** [comment]
Thanks!

**Pull request merged**